src/v3/webpack.dev.config.ts

```diff
@@ -82,13 +82,6 @@ const devConfig: Configuration = merge<Partial<Configuration>>(
     output: {
       path: `${PLAYGROUND}/target`,
     },
-    resolve: {
-      alias: {
-        // mock duo
-        '@okta/duo': resolve(PLAYGROUND, '/mocks/spec-duo/duo-mock.js'),
-        duo_web_sdk: resolve(__dirname, 'src/__mocks__/duo_web_sdk'),
```

Comment:
Removed this here because I am now passing an env flag to yarn command when executed for testcafe, this is to allow us to run the real version of DUO when testing locally. 

Another way to do this is to create a dedicated webpack config file specifically for testcafe, with a new yarn command/script, this way, no flag is required, and we can run the standard dev command to run the server locally and have the real DUO integration work. 

Looking for feedback on this approach.